src/bunit.web/Extensions/InputFile/InputFileExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿#if NET5_0_OR_GREATER
+using Microsoft.AspNetCore.Components.Forms;
+
+namespace Bunit;
+
+/// <summary>
+/// Extensions for the <see cref="InputFile"/> component.
+/// </summary>
+public static class InputFileExtensions
+{
+    /// <summary>
+    /// Uploads multiple files and invokes the OnChange event.
+    /// </summary>
+    /// <param name="inputFileComponent">The <see cref="InputFile"/> component which will upload the files.</param>
+    /// <param name="files">Files to upload.</param>
+    public static void UploadFiles(
+        this IRenderedComponent<InputFile> inputFileComponent,
+        params InputFileContent[] files)
+    {
+        if (inputFileComponent == null)
+            throw new ArgumentNullException(nameof(inputFileComponent));
+
+        if (!files.Any())
+            return;
+
+        var browserFiles = files.Select(f =>
+        {
+            var file = new BUnitBrowserFile
+            {
+                Name = f.Filename ?? string.Empty,
+                ContentType = f.ContentType ?? string.Empty,
+                LastModified = f.LastModified ?? default,
+                Content = f.Content,
+                Size = f.Size,
+            };
+
+            return file;
+        });
```

Comment:
```suggestion
        var browserFiles = files.Select(file => new BUnitBrowserFile
        {
            Name = file .Filename ?? string.Empty,
            ContentType = file.ContentType ?? string.Empty,
            LastModified = file.LastModified ?? default,
            Content = file.Content,
            Size = file.Size,
        });
```

I would propably just write it like this.